sw/airborne/modules/sensors/airspeed_sdp3x.c

```diff
@@ -91,9 +109,22 @@ PRINT_CONFIG_VAR(SDP3X_PRESSURE_OFFSET)
 #define SDP3X_AIRSPEED_SCALE 1.6327
 #endif
 
+/** Time constant for second order Butterworth low pass filter
+ * Default of 0.15 should give cut-off freq of 1/(2*pi*tau) ~= 1Hz
+ */
+#ifndef SDP3X_LOWPASS_TAU
+#define SDP3X_LOWPASS_TAU 0.15
+#endif
+
+#define SDP3X_SENDER_ID 42 //FIXME
```

Comment:
this should be a `AIRSPEED_SDP3X_ID` in the `subsystems/abi_sender_ids.h` file (and used to send ABI message)

Reply:
Added the ABI msg id in the airspeed msg